packages/core/test/specHelper.ts

```diff
@@ -506,3 +506,12 @@ export function interceptRequests() {
 export function isAdoptedStyleSheetsSupported() {
   return Boolean((document as any).adoptedStyleSheets)
 }
+
+export function isCSSStyleSheetConstructorSupported() {
+  try {
+    new CSSStyleSheet()
+  } catch (err) {
+    return false
+  }
+  return true
+}
```

Comment:
```suggestion
  return typeof CSSStyleSheet !== 'undefined'
}
```

Reply:
Sadly, this won't work because `CSSStyleSheet`  will be defined, it's a old API but the constructor is relatively recent, see: https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleSheet